tools/harness-thci/OpenThread_BR.py

```diff
@@ -423,7 +423,7 @@ def ping(self, strDestination, ilength=0, hop_limit=5, timeout=5):
         )
 
         self.bash(cmd)
-        time.sleep(1)
+        time.sleep(timeout)
```

Comment:
```suggestion
        time.sleep(min(5, timeout))
```
nit: avoid sleeping for too long?

Reply:
A timeout greater than 5 may be intentionally specified by the test script, I would leave it as is.